src/components/Navbar.tsx

```diff
@@ -0,0 +1,23 @@
+import React, {Dispatch, SetStateAction} from 'react';
+import MenuIcon from './MenuIcon';
+
+interface NavbarProps {
+  setToggleMenu: Dispatch<SetStateAction<boolean>>;
```

Comment:
do not pass setState as props, instead wrap it in a function and pass the function